plugins/catalog-common/package.json

```diff
@@ -44,6 +44,7 @@
     "clean": "backstage-cli package clean"
   },
   "dependencies": {
+    "@backstage/backend-common": "workspace:^",
```

Comment:
👍 to `@backstage/events`, although a little bit depending on exactly what we put there 😁. It might be the case that we should just add it to a different package, split it up into two separate packages like export from both `@backstage/core-plugin-api` and `@backstage/backend-plugin-api`, or create a new package but make it more generic.